ddtrace/contrib/dogpile_cache/__init__.py

```diff
@@ -0,0 +1,47 @@
+"""
```

Comment:
I noticed a small performance regression when testing more locally.

![Screenshot from 2019-11-01 16-21-30](https://user-images.githubusercontent.com/29210237/68061776-bfe83800-fcc3-11e9-8bae-5b9dbd6ae83a.png)

You can barely see it, but the blue (p95) for local API calls is a little higher (left side with tracing, right side without). Most likely bc of the overhead from all the lock/release tracing we're doing. I wanted to see how often locking issues cause performance problems. but we can drop that. It'd be nice to have a single span for all lock acquisition/release stuff, but dogpile's implementation doesn't make that easy or tenable

Reply:
I think I'll nuke the lock stuff. DD is way too slow with reviews and if they push back on this it'll probably take a billion more years for another review.